service/src/main/java/uk/gov/hmcts/reform/fpl/model/PartyExtended.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
+import uk.gov.hmcts.reform.fpl.model.common.Party;
+import uk.gov.hmcts.reform.fpl.model.common.TelephoneNumber;
+
+import java.util.Date;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
```

Comment:
Because base class contains some properties we should call methods in base class. Please set it to true.